djangocms_versioning/admin.py

```diff
@@ -467,13 +502,26 @@ def changelist_view(self, request, extra_context=None):
                 opts.app_label,
                 opts.model_name,
             )))
+        extra_context = extra_context or {}
+        versionable = versionables.for_content(self.model._source_model)
+        try:
+            grouper = versionable.get_grouper_with_fallbacks(
+                int(request.GET.get(versionable.grouper_field_name)),
+            )
+        except (ObjectDoesNotExist, TypeError, ValueError):
+            grouper = None
+        extra_context.update(
```

Comment:
should this be in else?
try
catch
else

Reply:
`get_grouper_with_fallbacks` can return None anyway, so no point in that. I could add a `if grouper:`.